src/main/java/org/mule/extension/objectstore/internal/ObjectStoreOperations.java

```diff
@@ -6,32 +6,12 @@
  */
 package org.mule.extension.objectstore.internal;
 
-import static java.lang.String.format;
-import static org.mule.extension.objectstore.internal.error.ObjectStoreErrors.INVALID_KEY;
-import static org.mule.extension.objectstore.internal.error.ObjectStoreErrors.KEY_ALREADY_EXISTS;
-import static org.mule.extension.objectstore.internal.error.ObjectStoreErrors.KEY_NOT_FOUND;
-import static org.mule.extension.objectstore.internal.error.ObjectStoreErrors.NULL_VALUE;
-import static org.mule.extension.objectstore.internal.error.ObjectStoreErrors.STORE_NOT_AVAILABLE;
-import static org.mule.runtime.api.i18n.I18nMessageFactory.createStaticMessage;
-import static org.mule.runtime.api.meta.ExpressionSupport.NOT_SUPPORTED;
-import static org.mule.runtime.api.meta.model.operation.ExecutionType.BLOCKING;
-import static org.mule.runtime.core.api.config.MuleProperties.OBJECT_STORE_MANAGER;
-import static org.mule.runtime.extension.api.error.MuleErrors.ANY;
-import org.mule.extension.objectstore.internal.error.AvailabilityErrorTypeProvider;
-import org.mule.extension.objectstore.internal.error.ContainsErrorTypeProvider;
-import org.mule.extension.objectstore.internal.error.RemoveErrorTypeProvider;
-import org.mule.extension.objectstore.internal.error.RetrieveErrorTypeProvider;
-import org.mule.extension.objectstore.internal.error.StoreErrorTypeProvider;
+import org.mule.extension.objectstore.internal.error.*;
 import org.mule.runtime.api.lock.LockFactory;
 import org.mule.runtime.api.meta.NamedObject;
 import org.mule.runtime.api.metadata.DataType;
 import org.mule.runtime.api.metadata.TypedValue;
-import org.mule.runtime.api.store.ObjectAlreadyExistsException;
-import org.mule.runtime.api.store.ObjectDoesNotExistException;
-import org.mule.runtime.api.store.ObjectStore;
-import org.mule.runtime.api.store.ObjectStoreException;
-import org.mule.runtime.api.store.ObjectStoreManager;
-import org.mule.runtime.api.store.ObjectStoreNotAvailableException;
+import org.mule.runtime.api.store.*;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Explicitly import the specific classes needed. ([squid:S2208](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2208))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-objectstore-connector&pullRequest=55&issues=AWnKcVnXotIXnmsUkTLj&open=AWnKcVnXotIXnmsUkTLj)

[ISSUE_KEY]: AWnKcVnXotIXnmsUkTLj